CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # Changelog
```

Comment:
I've aligned this file to the standard, that's why there are a couple of seemingly unrelated changes